back/src/domains/agency/use-cases/RegisterAgencyToInclusionConnectUser.ts

```diff
@@ -47,7 +47,11 @@ export class RegisterAgencyToInclusionConnectUser extends TransactionalUseCase<
     const agencyRights = await uow.agencyRepository.getAgenciesRightsByUserId(
       user.id,
     );
-    if (agencyRights.length > 0) {
+    const alreadyHasRequestedAgencyRight =
+      agencyRights.filter((agencyRight) =>
+        agencyIds.includes(agencyRight.agencyId),
+      ).length >= 1;
```

Comment:
```suggestion
      ).length;
```

Juste pour info c'est équivalent.